src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/ManageExternalAppointmentsService.kt

```diff
@@ -41,6 +41,7 @@ class ManageExternalAppointmentsService(
             startTime = appointment.startTime,
             endTime = appointment.endTime,
             internalLocationId = appointment.internalLocationId(),
+            extraInformation = appointment.extraInformation(),
```

Comment:
This is needed for VLB appointment types in activities and appointments.  It is simply free text so open to suggestions on what the value could look like.

Reply:
Ok for now - but this is where we would put the VLB comments.
I'm also thinking we should combine a) User comments + b) The court/probation team desc + c) The type of hearing/meeting + d) The video Url - so effectively construct a very meaningful comments text to show on prison appointments.  But this is for another ticket.  Just comments for now..